.env.template

```diff
@@ -1,10 +1,13 @@
 # Copy this file and store it as ".env" and fill in the required environment variables.
 DATA_DIR=U:/Research/FloodRiskResearch/DigitalTwin/stored_data
-DATA_DIR_REC=U:/Research/FloodRiskResearch/DigitalTwin/stored_data/rec_data
-DATA_DIR_MODEL_OUTPUT=U:/Research/FloodRiskResearch/DigitalTwin/stored_data/model_output
 DATA_DIR_GEOSERVER=U:/Research/FloodRiskResearch/DigitalTwin/stored_data/geoserver
 FLOOD_MODEL_DIR=U:/Research/FloodRiskResearch/DigitalTwin/BG-Flood/BG_Flood_v0-9
 
+USE_AWS_S3_BUCKET=False
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_BUCKET_NAME=
+
```

Comment:
So AWS S3 is optional right? That sounds like a good plan

Reply:
Yes, optional :)